libs/components/theme/src/lib/styles/_compat-tokens-mixins.scss

```diff
@@ -1,7 +1,19 @@
 @forward 'themes/modern/compat-tokens-mixins';
 
-@mixin sky-default-overrides($selector) {
-  #{$selector}:not(.sky-theme-modern *) {
+@mixin sky-default-overrides($selector: 'body') {
+  @if $selector == 'body' {
+    body:not(.sky-theme-modern) {
+      @content;
+    }
+  } @else {
```

Comment:
had to add this conditional logic because when `body` is the selector for some more global overrides it would select both default and modern, since the modern class is on the body so the `:not a child of modern` bit of the selector would technically be true for the body in modern